app/views/patients/show.html.erb

```diff
@@ -1,237 +1,266 @@
-          <td><% if assessment.symptomatic %>Yes<% else %>No<% end %></td>
-          <% end %>
-          <td><%= assessment.who_reported %></td>
-          <td><%= local_time(assessment.created_at, '%Y-%m-%d %H:%M %Z') %></td>
-          <% columns.each do |column| -%>
-          <% if assessment.get_reported_symptom_by_name(column) == nil %>
-          <td></td>
-          <% end %>
-          <%passes_threshold = assessment.symptom_passes_threshold(column)%>
-          <%symptom_value = assessment.get_reported_symptom_value(column)%>
-          <% if assessment.get_reported_symptom_by_name(column)&.type == "BoolSymptom"%>
-          <% if symptom_value == true %>
-          <% symptom_value = "Yes" %>
-          <% elsif symptom_value == false %>
-          <% symptom_value = "No"%>
-          <%end%>
-          <td><% if passes_threshold==true -%><span class="concern"><%=symptom_value%></span><% elsif passes_threshold==false -%><%=symptom_value%><%else -%><% end -%></td>
+            <%= react_component("subject/ClearReports", { authenticity_token: form_authenticity_token, patient: @patient_hash } ) %>
           <% end %>
-          <% if assessment.get_reported_symptom_by_name(column)&.type == "FloatSymptom" || assessment.get_reported_symptom_by_name(column)&.type == "IntegerSymptom"%>
-          <td><% if passes_threshold==true -%><span class="concern"><%=symptom_value%></span><% elsif passes_threshold==false -%><%=symptom_value%><%else -%><% end -%></td>
+          <% if @patient.monitoring %>
+            <%= react_component("subject/PauseNotifications", { authenticity_token: form_authenticity_token, patient: @patient_hash } ) %>
           <% end %>
-          <% end -%>
-          <td>
-            <div class="dropdown">
-              <button class="btn btn-sm btn-outline-primary dropdown-toggle btn-block a-dropdown" type="button" id="actionsButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                <i class="fas fa-cogs fw"></i>
-              </button>
-              <div class="dropdown-menu dropdown-menu-right" aria-labelledby="actionsButton">
-                <button type="button" data-toggle="modal" data-target="#modal-<%= assessment.id %>" class="dropdown-item"><i class="fas fa-edit fa-fw"></i> Edit</button>
-                <%= react_component("subject/AddReportNote", { assessment: assessment, patient: @patient, authenticity_token: form_authenticity_token }) %>
-                <% unless @patient.isolation %>
-                <%= react_component("subject/ClearSingleReport", { assessment_id: assessment.id, patient: @patient, authenticity_token: form_authenticity_token }) %>
+          <%= react_component("subject/ContactAttempt", { authenticity_token: form_authenticity_token, patient: @patient_hash } ) %>
+        </div>
+      </div>
+      <table class="assessment_table table table-sm table-striped table-bordered table-hover table-smaller-font" style="width:100%">
```

Comment:
probably not for the scope of this ticket, but were we thinking of/planning to maybe reactify these tables/finish removing DataTables in the future?

Reply:
Yeah, lower priority now though, since we've got the horizontal scroll enabled for these tables. Definitely not this PR, but a great point.